client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -10,6 +10,11 @@ import { useSelector } from 'react-redux';
 import './DataFilesSearchbar.module.css';
 
 const DataFilesSearchbar = ({ api, scheme, system, className }) => {
+  const err = useSelector(
```

Comment:
Perhaps we should rename this to `disabled` now with the inclusion of `loading`
```suggestion
  const disabled = useSelector(
```